music_service_async_interface/__init__.py

```diff
@@ -78,8 +78,8 @@ class Session(ABC):
     def __init__(
         self, obj_cls: Type["Object"], quality_cls: Type[AudioQuality], sess: Optional[aiohttp.ClientSession] = None
     ):
-        self.quality = quality_cls
-        self.obj = obj_cls
+        self.__class__.quality = quality_cls
+        self.__class__.obj = obj_cls
```

Comment:
What the heck happened here?

Add a simple check if `self.__class__.obj` is set.

Reply:
otherwise raise

Reply:
good idea, I didn't think about it